inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -288,20 +303,47 @@ MKLDNNPadNode::PadExecutor::PadExecutor(const PadAttrs& attrs,
     }
 }
 
+struct PadCornerContext {
+    void *dstPtr;
+    VectorDims dstDims;
+    float padValue;
+};
+
+template<typename T>
+struct PadCornerEmitter {
+    void operator()(PadCornerContext & ctx) {
+        const auto workAmount = std::accumulate(ctx.dstDims.begin(), ctx.dstDims.end(), 1, std::multiplies<size_t>());
+        T* data = reinterpret_cast<T*>(ctx.dstPtr);
+        parallel_for(workAmount, [&](size_t i) {
+            data[i] = static_cast<T>(ctx.padValue);
+        });
+    }
+};
+
 void MKLDNNPadNode::PadExecutor::exec(MKLDNNMemoryPtr& srcMemPtr, MKLDNNMemoryPtr& dstMemPtr) {
-    switch (params.attrs.padMode) {
-        case CONSTANT:
-            padConstant(srcMemPtr, dstMemPtr);
-            break;
-        case EDGE:
-            padEdge(srcMemPtr, dstMemPtr);
-            break;
-        case REFLECT:
-            padReflectOrSymmetric(srcMemPtr, dstMemPtr);
-            break;
-        case SYMMETRIC:
-            padReflectOrSymmetric(srcMemPtr, dstMemPtr, true);
-            break;
+    if (cornerCase) {
+        PadCornerContext ctx { dstMemPtr->GetPtr(), dstMemPtr->getStaticDims(), params.attrs.padValue };
+        OV_SWITCH(MKLDNNPlugin, PadCornerEmitter, ctx, params.attrs.prc,
```

Comment:
Can we use memset to avoid this switch case?

Reply:
now we call `padConstant` for corner case